packages/terra-site/src/examples/embedded-content-consumer/consumers/CustomEventsConsumer.jsx

```diff
@@ -21,7 +21,6 @@ const CustomEventsConsumer = () => (
     <EmbeddedContentConsumer
       src="#/tests/embedded-content-consumer-tests/custom-events-provider"
       eventHandlers={eventHandlers}
-      fill
```

Comment:
Did we have a reason for these props? I couldn't find any documentation on them, and I found leaving them off still resulted in passing tests, and visual regression didn't reveal any noticeable behavior.